backend/ttrs/views.py

```diff
@@ -20,16 +32,62 @@ class StudentDetail(generics.RetrieveUpdateDestroyAPIView):
     serializer_class = StudentSerializer
     permission_classes = (IsAuthenticated, IsTheStudent)
 
-    def update(self, request, *args, **kwargs):
```

Comment:
the `update` method turned out to be unnecessary because the code is completely same as super class.